### ISSUE TITLE
Bug: appimage crash on startup

### ISSUE THREAD

**Issue by b4n6-b4n6** [cypherstack/stack_wallet#1042]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I open appimage - it crashes

### Expected Behavior

appimage dosen't crash

### Reproduce Steps

open app via terminal

### Environment

- Operating system and version: Ubuntu 22.04.5 LTS
- Device platform and version: amd64
- Real device or emulator/simulator: N/A
- Stackwallet version: last 4 versions

### Further Information

logs before crash

```
flutter: Log: [Info][2024-12-03 23:32:12.092Z]: Logs older than 2024-11-03 23:32:12.089843Z cleared!
flutter: Log: [Info][2024-12-03 23:32:12.129Z]: Updating default light theme...
flutter: Log: [Info][2024-12-03 23:32:12.201Z]: Updating default light theme... finished
flutter: Log: [Info][2024-12-03 23:32:12.201Z]: Updating default dark theme...
flutter: Log: [Info][2024-12-03 23:32:12.225Z]: Updating default dark theme... finished
flutter: BUILD: _MaterialAppWithThemeState
malloc(): unsorted double linked list corrupted
Aborted (core dumped)
```

so exactly one time it opened and I created some wallets, after restarting my machine and running stackwallet again I get new error

```
flutter: Log: [Info][2024-12-03 23:21:59.798Z]: Logs older than 2024-11-03 23:21:59.796640Z cleared!
flutter: BUILD: _MaterialAppWithThemeState
double free or corruption (!prev)
Aborted (core dumped)
```

**Comment by RokeJulianLockhart** [comment]
> https://github.com/cypherstack/stack_wallet/issues/1042#issue-2716209250

@b4n6-b4n6, is this a duplicate of https://github.com/cypherstack/stack_wallet/issues/1021#issue-2663121474?

**Comment by julian-CStack** [comment]
@b4n6-b4n6 I believe I've got a fix for the library causing the crash and I've opened a PR. If I don't hear anything back in the next bit I may point stack wallet at our clone of the repo with the fix.

For now there is a temporary fix mentioned in #1021 
If you end up trying that out and it doesn't work please let me know.

**Comment by b4n6-b4n6** [comment]
> @b4n6-b4n6 I believe I've got a fix for the library causing the crash and I've opened a PR. If I don't hear anything back in the next bit I may point stack wallet at our clone of the repo with the fix.
> 
> For now there is a temporary fix mentioned in #1021 If you end up trying that out and it doesn't work please let me know.

the culprit has been identified as LC_ALL in my environment - setting it to en_US.UTF-8 solved the issue
closing ticket
